src/Sentry/Internal/SentryScopeManager.cs

```diff
@@ -86,13 +86,13 @@ public void WithScope(Action<Scope> scopeCallback)
             using (PushScope())
             {
                 var scope = GetCurrent();
-                scopeCallback?.Invoke(scope.Key);
+                scopeCallback.Invoke(scope.Key);
```

Comment:
Here I'm afraid `net461` apps might pass a null reference accidentally. That first point we discussed about null annotation vs defensive programming